packages/in_app_purchase/in_app_purchase_ios/ios/in_app_purchase.podspec

```diff
@@ -0,0 +1,27 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+Pod::Spec.new do |s|
+  s.name             = 'in_app_purchase'
+  s.version          = '0.0.1'
+  s.summary          = 'Flutter In App Purchase'
+  s.description      = <<-DESC
+A Flutter plugin for in-app purchases. Exposes APIs for making in-app purchases through the App Store.
+Downloaded by pub (not CocoaPods).
+                       DESC
+  s.homepage         = 'https://github.com/flutter/plugins'
+  s.license          = { :type => 'BSD', :file => '../LICENSE' }
+  s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
+  s.source           = { :http => 'https://github.com/flutter/plugins/tree/master/packages/in_app_purchase' }
+  s.documentation_url = 'https://pub.dev/packages/in_app_purchase'
```

Comment:
check

Reply:
note that this is causing linter warnings in the tree since the url does not exists yet:

```
url: The URL (https://pub.dev/packages/in_app_purchase_ios) is not reachable.
```

not sure how serious we should treat this?